src/main/java/hw_17/Test1.java

```diff
@@ -0,0 +1,34 @@
+package hw_17;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+import org.junit.Assert;
+
+
+public class Test1 {
+    public static void main(String[] args) {
```

Comment:
Test is always failed
![image](https://github.com/user-attachments/assets/ecccf186-6140-4fb7-930b-f12eb6e0af60)

Reply:
Just remove line
![Screenshot_1](https://github.com/user-attachments/assets/b4b43070-38b9-460e-a13d-70f8f2683c86)

Reply:
just remove this line
![Screenshot_1](https://github.com/user-attachments/assets/9d2a4d0e-76fd-4c9e-80bd-69ec0f08059f)

Reply:
done